src/libstore/daemon.cc

```diff
@@ -1091,7 +1090,7 @@ void processConnection(
             debug("performing daemon worker op: %d", op);
 
             try {
-                performOp(tunnelLogger, store, trusted, recursive, clientVersion, from, to, op);
+                performOp(tunnelLogger, store, trusted, recursive, clientVersion, conn.from, conn.to, op);
```

Comment:
I think we might as well do this simplificaiton now :)
```suggestion
                performOp(tunnelLogger, store, trusted, recursive, conn, op);
```